docs/data-sources/node.md

```diff
@@ -50,32 +50,32 @@ data "wallarm_node" "example" {
-- `enabled` - (Optional) Indicator of the WAF node status. Can be: `true` for enabled WAF nodes and `false` for disabled WAF nodes.
+- `uuid` - (optional) Wallarm node UUID.
+- `hostname` - (optional) Wallarm node name.
+- `type` - (optional) Wallarm node type. Can be: `cloud_node` for cloud Wallarm nodes, `node` for regular Wallarm nodes.
+- `enabled` - (optional) indicator of the Wallarm node status. Can be: `true` for enabled Wallarm nodes and `false` for disabled Wallarm nodes.
 
-To get details on all created WAF nodes, specify an empty set of the filters (`filter {}`).
+To get details on all created Wallarm nodes, specify an empty set of the filters (`filter {}`).
 
 ## Attributes Reference
 
-`nodes` - WAF node attributes in the `key=value` format. Possible keys:
-
-- `id` - Internal WAF node ID.
-- `hostname` - WAF node name.
-- `type` - WAF node type. Can be: `cloud_node` for cloud WAF nodes, `node` for regular WAF nodes.
-- `uuid` - WAF node UUID.
-- `enabled` - Indicator of the WAF node status. Can be: `true` for enabled WAF nodes and `false` for disabled WAF nodes.
-- `client_id` - ID of the client installed the WAF node.
-- `active` - Node syncronisation status
-- `instance_count` - Number of instances with the installed WAF node. Only for the `cloud_node` type.
-- `active_instance_count` - Number of active instances with the installed WAF node. Only for the `cloud_node` type.
-- `token` - WAF node token. Only for the `cloud_node` type.
-- `requests_amount` - Number of requests processed by the WAF node.
-- `ip` - IP address of the WAF node at the time of the last synchronization.
-- `proton` - Installed `proton.db` version. Can be an integer value or `null`.
-- `lom` - Installed `lom` version. Can be an integer value or `null`.
+`nodes` - Wallarm node attributes in the `key=value` format. Possible keys:
+
+- `id` - internal Wallarm node ID.
+- `hostname` - Wallarm node name.
+- `type` - Wallarm node type. Can be: `cloud_node` for cloud Wallarm nodes, `node` for regular Wallarm nodes.
+- `uuid` - Wallarm node UUID.
+- `enabled` - indicator of the Wallarm node status. Can be: `true` for enabled Wallarm nodes and `false` for disabled Wallarm nodes.
+- `client_id` - ID of the client installed the Wallarm node.
+- `active` - node synchronization status
+- `instance_count` - number of instances with the installed Wallarm node. Only for the `cloud_node` type.
+- `active_instance_count` - number of active instances with the installed Wallarm node. Only for the `cloud_node` type.
+- `token` - Wallarm node token. Only for the `cloud_node` type.
+- `requests_amount` - number of requests processed by the Wallarm node.
+- `ip` - IP address of the Wallarm node at the time of the last synchronization.
+- `proton` - installed `proton.db` version. Can be an integer value or `null`.
+- `lom` - installed `lom` version. Can be an integer value or `null`.
```

Comment:
mention a correct term here as well, otherwise some customers using the new term can be confused